lgc/patch/SystemValues.h

```diff
@@ -162,12 +162,9 @@ class ShaderSystemValues {
 
   llvm::SmallVector<llvm::Value *, 8> m_descTablePtrs;       // Descriptor table pointers
   llvm::SmallVector<llvm::Value *, 8> m_shadowDescTablePtrs; // Shadow descriptor table pointers
-  llvm::Value *m_internalGlobalTablePtr = nullptr;
-  // Internal global table pointer
-  llvm::Value *m_internalPerShaderTablePtr = nullptr;
-  // Internal per shader table pointer
-  llvm::Value *m_spilledPushConstTablePtr = nullptr;
-  // Spilled push constant pointer
+  llvm::Instruction *m_internalGlobalTablePtr = nullptr;     // Internal global table pointer
+  llvm::Value *m_internalPerShaderTablePtr = nullptr;        // Internal per shader table pointer
```

Comment:
Internal per-shader table is not used in LLPC. Maybe we can skip it now. If I remember correctly, it is still added as user_data_1 because PAL once expected us to provide it. I think you can make a separate change to have a try. If no CTS fails, it is safe to remove it and make s1 available to other uses.

Reply:
Good idea for a future change. Presumably we still need to start in s2 in compute shaders.